### ISSUE TITLE
How can we open the Multi version support

### ISSUE THREAD

**Issue by Liuyiyang888888** [Moresteck/Project-Poseidon-Uberbukkit#42]

**Comment by Moresteck** [comment]
Can you elaborate on what you mean? What's the issue exactly?

**Comment by Unknown888888Name** [comment]
The Muti version support doesn’t work

**Comment by notdevcody** [comment]
> The Muti version support doesn’t work

What do you mean **exactly**? Can players not join from multiple versions? Does the server crash? Also provide the uberbukkit version you’re using along with the configuration.

**Comment by Unknown888888Name** [comment]
Yes, players **can’t** join from a version other than the server version.
**The server didn't crash!It only says “End of stream”**
;(

> The server version is 

[**2.0.2**]
[https://github.com/Moresteck/Project-Poseidon-Uberbukkit/releases/tag/2.0.2](url)

> Commit version:

**5896447**
[https://github.com/Moresteck/Project-Poseidon-Uberbukkit/commit/58964471ddfeeede4fcd86a2552d94732dd9940a](url)

**Comment by Unknown888888Name** [comment]
The [https://github.com/Moresteck/Project-Poseidon-Uberbukkit/blob/9784e6138a5e0fe7dd3ce1366da43b5d21294e4f/README.md](url) says:
Multi version support: UberBukkit supports b1.0 - b1.7.3 versions.
Experimental alpha support is also available.

**Comment by Moresteck** [comment]
Please provide your poseidon.yml and uberbukkit.yml config.

**Comment by Unknown888888Name** [comment]
OKAY
[poseidon.yml.TXT](https://github.com/user-attachments/files/18305866/poseidon.yml.TXT)
[uberbukkit.yml.TXT](https://github.com/user-attachments/files/18305867/uberbukkit.yml.TXT)

**Comment by Unknown888888Name** [comment]
![image](https://github.com/user-attachments/assets/aa4cd442-8789-451f-a002-c63a0f13d38d)

**Comment by Unknown888888Name** [comment]
![image](https://github.com/user-attachments/assets/2b6ad17a-db18-46d0-84b0-e9ba7964b606)

**Comment by Unknown888888Name** [comment]
emm

**Comment by Moresteck** [comment]
You configured your uberbukkit.yml for b1.7.3, not for b1.2_02 or a1.2.6.

**Comment by Unknown888888Name** [comment]
Please tell me what should I do.I don't know how to fix this.

**Comment by notdevcody** [comment]
Grab a pre-made configuration from [here](https://github.com/Moresteck/uberbukkit/tree/develop/defaults) and rename it to uberbukkit.yml.

**Comment by Unknown888888Name** [comment]
@notdevcody oh thanks so much!
